### PR TITLE
Remove newlines in url titles

### PR DESCRIPTION
**Pull request by jamesfielder**
- Strip newlines
- Add new npm run fix script to fix indentation with eslint

### CODE REVIEW

**Comment by jamesfielder** [comment]
Fix for #3

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint ./src ./test",
+    "fix": "eslint --fix ./src ./test",
```
Comment (nullren) [inline package.json]: is there a rule where we can specify long lines and how to treat them?

File: src/plugins/url-titles.js [hunk 2]
```diff
@@ -14,7 +14,9 @@ module.exports = (client, request) =>
         }
 
         const $ = cheerio.load(body)
-        const title = $('title').text().trim()
+        const title = $('title').text()
+                      .replace(/\s+/g, ' ')
+                      .trim()
```
Comment (nullren) [inline src/plugins/url-titles.js]: for example was this "fixed" by eslint?

File: test/unit/plugins/url-titles.js [hunk 3]
```diff
@@ -61,4 +61,29 @@ describe('url-titles', () => {
...
+      addListener: (name, callback) => {
+        callback('you', 'me', 'message containing url https://google.com :D')
+      },
+      say: sinon.spy(),
+      nick: 'me'
+    }
+
+    require('../../../src/plugins/url-titles.js')(client, request)
+
+    client.say.should.have.been.calledWith('you', '>>> lol who put all these newlines in? How silly.')
```
Comment (nullren) [inline test/unit/plugins/url-titles.js]: and should this have been "fixed"?

Comment (jamesfielder) [inline test/unit/plugins/url-titles.js]: I was debating this as well. I didn't want to split it as I was a little worried that splitting it over new lines might actually introduce them into the test? I wasn't sure anyway. I appreciate it's a bit long, but I suppose that's what it's testing that the output will be. Suggestions welcomed!

File: package.json (hunk 1 above)
Comment (jamesfielder) [inline package.json]: I suppose we could set this: http://eslint.org/docs/rules/max-len

**Pull request merged**